.eslintrc

```diff
@@ -2,7 +2,39 @@
     "extends": "girder",
     "rules": {
         "no-throw-literal": "off",
+        "backbone/defaults-on-top": "error",
```

Comment:
This is only a staging ground, since it's much easier to discuss and apply rules locally here. Once this is approved, I'll generate another PR to `eslint-config-girder`, merge that, release that, update the dependency here in Girder's `package.json`, then remove these explicit local rules.